Index: build/toolchain.gypi
===================================================================
--- build/toolchain.gypi
+++ build/toolchain.gypi
@@ -519,6 +519,7 @@
             OS=="qnx"', {
             'cflags': [ '-Wall', '<(werror)', '-W', '-Wno-unused-parameter',
                         '-Wnon-virtual-dtor', '-Woverloaded-virtual',
+                        '-Wno-unused-local-typedefs', '-Wno-aggressive-loop-optimizations',
                         '<(wno_array_bounds)',
                       ],
             'conditions': [
Index: build/standalone.gypi
===================================================================
--- build/standalone.gypi
+++ build/standalone.gypi
@@ -173,6 +173,7 @@
        or OS=="netbsd"', {
       'target_defaults': {
         'cflags': [ '-Wall', '<(werror)', '-W', '-Wno-unused-parameter',
+                    '-Wno-unused-local-typedefs', '-Wno-aggressive-loop-optimizations',
                     '-pthread', '-fno-exceptions', '-pedantic' ],
         'cflags_cc': [ '-Wnon-virtual-dtor', '-fno-rtti' ],
         'ldflags': [ '-pthread', ],
